src/main/java/org/opentripplanner/standalone/config/routerequest/TriangleOptimizationConfig.java

```diff
@@ -0,0 +1,54 @@
+package org.opentripplanner.standalone.config.routerequest;
+
+import static org.opentripplanner.standalone.config.framework.json.OtpVersion.V2_0;
+import static org.opentripplanner.standalone.config.framework.json.OtpVersion.V2_5;
+
+import org.opentripplanner.routing.api.request.preference.TimeSlopeSafetyTriangle;
+import org.opentripplanner.standalone.config.framework.json.NodeAdapter;
+
+public class TriangleOptimizationConfig {
+
+  private static void mapTriangleParameters(
```

Comment:
The main method/entry point of a class should be the first method, `private` methods go down at the bottom of the class.